src/unittest/python/test_cli.py

```diff
@@ -0,0 +1,204 @@
+# Copyright (c) 2020 Intel Corporation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#      http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from mock import patch
+from mock import Mock
+from mock import MagicMock
+from datetime import datetime
+
+
+from dha.cli import get_parser
+from dha.cli import configure_logging
+from dha.cli import write_csv
+from dha.cli import get_image_count
+from dha.cli import get_all_images
+from dha.cli import filter_image_list
+from dha.cli import filter_tag_list
+from dha.cli import get_all_tags
+from dha.cli import get_tags
+from argparse import ArgumentParser
+
+
+class TestCLI(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    @patch('dha.cli.ArgumentParser.add_argument')
+    @patch('dha.cli.ArgumentParser.parse_args')
+    def test__get_args_Should_ReturnExpected_When_Called(self,
+                                                         parse_args_patch,
+                                                         *patches):
+        parse_args_patch.return_value = 'parsed args'
+        result = get_parser()
+        self.assertIsInstance(result, ArgumentParser)
+
+    @patch('dha.cli.logging')
+    def test__configure_logging_Should_CallExpected_When_Called(self,
+                                                                logging_patch,
+                                                                *patches):
+        args = Mock()
+        root_logger_mock = Mock()
+        logging_patch.getLogger.return_value = root_logger_mock
+        configure_logging(args)
+        root_logger_mock.addHandler.assert_called_once()
+
+    @patch('dha.cli.logging')
+    def test__configure_logging_Should_CallExpected_When_Called_With_Args(
+        self,
+        logging_patch,
+            *patches):
+        args = Mock()
```

Comment:
indentation is not distinguishable:
https://www.python.org/dev/peps/pep-0008/#indentation